### ISSUE TITLE
Discuss about GDPR and CCPA compliance

### ISSUE THREAD

**Issue by Vyvy-vi** [commons-stack/praise#474]
The current version of the system doesn't take these into account.
We might have to start looking more into these in terms of problems that can harm us in the long-term, esp. as the number of instances hosted by communities increases.

It can be a problem for certain orgs that happen to have a legal body or business attached to their community. Their praise instance would need to be GDPR-compliant.

**Comment by kristoferlund** [comment]
Yeah. Big interesting and difficult topic. Transparency on the one hand, right to privacy in the other. How could we build a system that fulfills both needs?

- Rewards payments would have to use ZK proofs so payments cannot be connected to Praise receivers.
- The Praise data can definitely be sensitive information. We would like each user to be able to control access to that individual praise data. The data would have to be encrypted by default, the individual user granting access to the Praise system or third party applications.
